portway/portway
# Swapping for EasyMDE fork

* Swaps out SimpleMDE for EasyMDE, an actively maintained fork
* This is so we can not have XSS vulnerability

**To test**

Run `make installweb` and edit markdown in a new document or existing document. Should behave like before.
